colossalai/pipeline/schedule/one_f_one_b.py

```diff
@@ -285,5 +296,6 @@ def forward_backward_step(self,
                 self.send_backward(input_obj_grad)
 
         if outputs is not None:
-            outputs = merge_batch(outputs)
+            bsz_dim = 1 if model.unwrap().__class__.__name__ in CHATGLM_MODELS else 0
```

Comment:
Don't judge model name or class. Set this attribute to model and use `getattr()` here.

Reply:
DONE

Reply:
move this attribute at `configure`